File: app/src/main/java/com/example/plantea/presentacion/EventoInterface.java
package com.example.plantea.presentacion;

import com.example.plantea.dominio.Evento;

public interface EventoInterface {

    //EventosFragment
    void crearEventoFragment();

    //NuevoEventoFragment
    void nuevoEvento(Evento evento);
    void planificar();
    void cancelarEvento();
    void cancelarNotificacion(int identificador);

}

File: app/src/main/java/com/example/plantea/presentacion/EventoInterface.kt
package com.example.plantea.presentacion

import android.content.Context
import com.example.plantea.dominio.Evento

interface EventoInterface {
    //EventosFragment
    fun crearEventoFragment(context: Context)

    //NuevoEventoFragment
    fun nuevoEvento(context: Context, evento: Evento)
    fun planificar(context: Context)
    fun cancelarEvento(context: Context)
    fun cancelarNotificacion(context: Context, identificador: Int)

    fun clickReloj(tiempo: CharSequence?)
}